services/media/service/src/ingest/plugins/start-ingest-endpoint-plugin.ts

```diff
@@ -118,16 +118,26 @@ export const StartIngestEndpointPlugin = makeExtendSchemaPlugin((build) => {
             const ajv = new Ajv({ allErrors: true });
             addFormats(ajv);
             const isValid = ajv.validate(ingestSchema, document);
-            const customErrors = customIngestValidation(document);
-            if (!isValid || customErrors.length > 0) {
+            if (!isValid) {
               const schemaErrors = transformAjvErrors(
                 ajv.errors,
                 documentString,
               );
               throw new MosaicError({
                 ...CommonErrors.IngestValidationError,
                 details: {
-                  validationErrors: [...schemaErrors, ...customErrors],
+                  validationErrors: [...schemaErrors],
+                },
+              });
+            }
+            // Skip custom errors if schema validation failed, since incorrect
```

Comment:
Not sure about the place for this comment - would it logically make more sense in line 126 where you throw the error? I read this comment and assumed it was a comment for line 135/136 and was wondering how that is skipped :-)